### PR TITLE
Messages & layout

### PR DESCRIPTION
**Pull request by levymetal**
Here we've got messages for jobs, a 100% height layout, and a wip for job details

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.2.2
```
Comment (ashfogel) [inline .ruby-version]: Hey Christian, I have specified the ruby version being used up on Heroku in the Gemfile. Can you please remove this file? thanks!

File: web/test/controllers/jobs_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
```
Comment (ashfogel) [inline web/test/controllers/jobs_controller_test.rb]: Can you please also remove this scaffolded controller test?

**Comment by ashfogel** [comment]
hey Christian, I have left some notes on two files that we don't need, otherwise everything looks great, can you please remove those two files and resubmit the PR, thanks.

File: web/test/controllers/jobs_controller_test.rb (hunk 2 above)
Comment (levymetal) [inline web/test/controllers/jobs_controller_test.rb]: rbenv added this file when I specified 2.2.2 as a local for this app. We could remove it then gitignore it, but having said that, wouldn't it be a good idea to keep it in the repo to ensure everyone is always running the same environment?



—

**Pull request closed without merging**